src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,163 @@
+package duke.task;
+
+import duke.exception.EmptyDescriptionException;
+import duke.exception.InvalidCommandException;
+import duke.exception.LoadDataException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskManager {
+
+    private final ArrayList<Task> tasks = new ArrayList<>();
+
+    public String getData() {
+        StringBuilder data = new StringBuilder();
+        for (Task task : tasks) {
+            data.append(task.getData()).append(System.lineSeparator());
```

Comment:
Try to avoid complicated/nested expression. Split them to multiple lines for better readability.